<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lscom</class>
 <widget class="QMainWindow" name="lscom">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>620</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>lscom</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>380</y>
      <width>231</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>串口号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comPortBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Parity">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_BaudRate">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_StopBits">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_DateBits">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>奇偶校验</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>停止位</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btu_send_data">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>发  送</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="log_text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1080</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_send">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>661</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btu_send_data_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>清除窗口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu_send_data_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_file_path">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btu_send_data_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>340</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>发送文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu_send_data_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>340</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu_send_data_6">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>340</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清发送区</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>340</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>最前</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>340</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>English</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu_send_data_7">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>340</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存参数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu_send_data_8">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>340</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>扩展</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu_send_data_9">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>340</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>——</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu_open_com">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btu_close_com">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭端口</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX显示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu_send_data_10">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>370</y>
      <width>92</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>保存数据</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>370</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>接收数据到文件</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX发送</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
